### PR TITLE
Remove bitcoin_support dependency from blockchain_contracts

### PR DESCRIPTION
**Pull request by D4nte**
Related to coblox/blockchain_contracts#4

Note: Issue to be open to contribute the TODOs back, see coblox/blockchain_contracts#4.

### CODE REVIEW

File: vendor/bitcoin_support/src/network.rs [hunk 1]
```diff
@@ -71,7 +71,7 @@ mod test {
         assert_eq!(testnet, "test");
     }
 
-    fn assert_display<T: Display>(t: T) {}
+    fn assert_display<T: Display>(_t: T) {}
```
Comment (luckysori) [inline vendor/bitcoin_support/src/network.rs]: :mega: I guess you shouldn't fix this unless you want to do #1139 too.

File: vendor/ethereum_support/src/network.rs [hunk 2]
```diff
@@ -71,7 +71,7 @@ mod test {
         );
     }
 
-    fn assert_display<T: Display>(t: T) {}
+    fn assert_display<T: Display>(_t: T) {}
```
Comment (luckysori) [inline vendor/ethereum_support/src/network.rs]: :mega: How dare you fix Clippy warnings?!!

File: vendor/bitcoin_support/src/network.rs (hunk 1 above)
Comment (thomaseizinger) [reply vendor/bitcoin_support/src/network.rs]: It sounds like a nitpick but the ticket also includes to investigate why clippy doesn't fail this so it should probably not be done as part of this :)

**Comment by D4nte** [comment]
> Might be confusing to fix those warnings here without dealing with #1139 in full, but approved nevertheless.

Good point, I'll trash this commit, rebase and merge.

**Pull request merged**